src/cli/serve/integration_tests/invalid_config.test.ts

```diff
@@ -31,41 +32,27 @@ describe('cli invalid config support', function () {
           cwd: REPO_ROOT,
         }
       );
+      expect(error).toBe(undefined);
 
-      let fatalLogLine;
+      let fatalLogEntries;
       try {
-        [fatalLogLine] = stdout
+        fatalLogEntries = stdout
           .toString('utf8')
           .split('\n')
           .filter(Boolean)
           .map((line) => JSON.parse(line) as LogEntry)
-          .filter((line) => line.tags?.includes('fatal'))
-          .map((obj) => ({
-            ...obj,
-            pid: '## PID ##',
-            '@timestamp': '## @timestamp ##',
-            error: '## Error with stack trace ##',
-          }));
+          .filter((line) => line.log.level === 'FATAL');
```

Comment:
I initially tried to adapt the test to use the `pattern` layout and search for lines containing `[FATAL]`, but the test was failing on CI and I couldn't determine why (`highligh` is `true` by default, maybe when running on CI the highlighting get enabled and cause the strings to not match) So I adapted `src/cli/serve/integration_tests/__fixtures__/invalid_config.yml` instead to use the `json` appender, which is closer to the initial test anyway.